hail/python/hail/experimental/datasets.json

```diff
@@ -1581,5 +1581,311 @@
         "version": "1.1"
       }
     ]
+  },
+  "panukb_ld_block_matrix_AFR": {
+    "description": "Pan-UKB: linkage disequilibrium (LD) matrix Hail BlockMatrix for African ancestry population.",
+    "url": "https://pan.ukbb.broadinstitute.org/docs/technical-overview",
+    "versions": [
+      {
+        "reference_genome": "GRCh37",
+        "url": {
+          "aws": {
+            "us": "s3://pan-ukb-us-east-1/ld_release/UKBB.AFR.ldadj.bm"
+          }
+        },
+        "version": "0.2"
+      }
+    ]
+  },
+  "panukb_ld_block_matrix_AMR": {
+    "description": "Pan-UKB: linkage disequilibrium (LD) matrix Hail BlockMatrix for Admixed American ancestry population.",
+    "url": "https://pan.ukbb.broadinstitute.org/docs/technical-overview",
+    "versions": [
+      {
+        "reference_genome": "GRCh37",
+        "url": {
+          "aws": {
+            "us": "s3://pan-ukb-us-east-1/ld_release/UKBB.AMR.ldadj.bm"
+          }
+        },
+        "version": "0.2"
+      }
+    ]
+  },
+  "panukb_ld_block_matrix_CSA": {
+    "description": "Pan-UKB: linkage disequilibrium (LD) matrix Hail BlockMatrix for Central/South Asian ancestry population.",
+    "url": "https://pan.ukbb.broadinstitute.org/docs/technical-overview",
+    "versions": [
+      {
+        "reference_genome": "GRCh37",
+        "url": {
+          "aws": {
+            "us": "s3://pan-ukb-us-east-1/ld_release/UKBB.CSA.ldadj.bm"
+          }
+        },
+        "version": "0.2"
+      }
+    ]
+  },
+  "panukb_ld_block_matrix_EAS": {
+    "description": "Pan-UKB: linkage disequilibrium (LD) matrix Hail BlockMatrix for East Asian ancestry population.",
+    "url": "https://pan.ukbb.broadinstitute.org/docs/technical-overview",
+    "versions": [
+      {
+        "reference_genome": "GRCh37",
+        "url": {
+          "aws": {
+            "us": "s3://pan-ukb-us-east-1/ld_release/UKBB.EAS.ldadj.bm"
+          }
+        },
+        "version": "0.2"
+      }
+    ]
+  },
+  "panukb_ld_block_matrix_EUR": {
+    "description": "Pan-UKB: linkage disequilibrium (LD) matrix Hail BlockMatrix for European ancestry population.",
+    "url": "https://pan.ukbb.broadinstitute.org/docs/technical-overview",
+    "versions": [
+      {
+        "reference_genome": "GRCh37",
```

Comment:
Block Matrices don't have a reference genome

Reply:
In the description for the Block Matrices I can add something like "see associated Hail Table x". And you want that to refer to the associated variant index HT, correct?

Reply:
Yep, that would be awesome!

Reply:
Sounds good! I just added that reference to the BM descriptions.